sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py

```diff
@@ -770,6 +897,35 @@ def _joint_states_callback(self, joint_state):
             self._joints_effort = {n: v for n, v in
                                    zip(joint_state.name, joint_state.effort)}
 
+    def _set_up_set_points_subscribers(self, controllers_list):
+        """
+        Sets up the required subscribers to read from the set points of each given controller
+        @param controller_list - Dictionary mapping a trajectory controller with the list of the joints it has
+        """
+        # Get joint names of the group
+        joint_names_group = self._move_group_commander.get_active_joints()
+        topics_subscribed = []
+
+        for controller_name in controllers_list.keys():
+            for joint_name in joint_names_group:
+                if joint_name in controllers_list[controller_name]:
+                    topic_name = f"/{controller_name}/state"
+                    if topic_name not in topics_subscribed:
+                        rospy.Subscriber(topic_name, JointTrajectoryControllerState, self._set_point_cb, queue_size=1)
+                        topics_subscribed.append(topic_name)
+
+        for joint_name in joint_names_group:
+            joint_member = joint_name[3:-2]
+            joint_number = joint_name[5:]
```

Comment:
Add as comment the structure of joint_name, or use any() in the if below if you can :)

Reply:
Maybe a regex with a substring match?